.github/workflows/ci.yml

```diff
@@ -15,3 +15,5 @@ jobs:
     needs: Spec
     uses: "puppetlabs/cat-github-actions/.github/workflows/module_acceptance.yml@main"
     secrets: "inherit"
+    with:
+      runs_on: "ubuntu-20.04"
```

Comment:
why do we need this old image? Can't we use the stable ubuntu-22.04?

Reply:
@bastelfreak
The EL-7 based docker images are unable to be provisioned on ubuntu-2204 in nightly tests: https://github.com/puppetlabs/puppetlabs-concat/actions/runs/11395101272. I think we may need to pin to ubuntu-2004 till we remove those platforms.